src/query.md

```diff
@@ -292,8 +276,6 @@ rustc_queries! {
 }
```

Comment:
this still looks outdated, and a bit redundant with the first section - can we remove it and add `desc` at the top?

Reply:
given that `desc` is required this makes a lot of sense